### PR TITLE
support loadtest users in new restore code

### PR DESCRIPTION
**Pull request by czue**

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/phone/data_providers/case/load_testing.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from copy import deepcopy
+from casexml.apps.case.models import CommCareCase
+from casexml.apps.case.xml import V2
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/phone/data_providers/case/load_testing.py]: F401 'V2' imported but unused

File: corehq/ex-submodules/casexml/apps/phone/data_providers/case/load_testing.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    """
+    current_count = 0
+    original_update = update
+    while current_count < restore_state.loadtest_factor:
+        element = get_case_element(update.case, update.required_updates, restore_state.version)
+        response.append(element)
+        current_count += 1
+        if current_count < restore_state.loadtest_factor:
+            update = transform_loadtest_update(original_update, current_count)
+
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/phone/data_providers/case/load_testing.py]: W391 blank line at end of file

**Comment by czue** [comment]
mostly just moving stuff around. wait for tests. @snopoke @NoahCarnahan

**Comment by snopoke** [comment]
:white_check_mark: once tests pass

**Comment by czue** [comment]
(passed)

**Pull request merged**